app/synchronizers/bookingsync_portal/write/source.rb

```diff
@@ -0,0 +1,30 @@
+class BookingsyncPortal::Write::Source
+  attr_reader :account
+  private :account
+
+  def initialize(account)
+    @account = account
+  end
+
+  def synchronize
+    find_or_create
+  end
+
+  private
+
+  def find_or_create
+    find_source || create_source
+  end
+
+  def api
+    account.api
+  end
+
+  def find_source
+    api.sources.find { |s| s.name == BookingsyncPortal.source_name }
```

Comment:
This is not good, we need to be more relaxed on this and allow wrong capitalization